resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -640,6 +640,7 @@ s3.new.folder.name=Folder name:
 s3.open.file_too_big=The editor cannot open files larger than {0}
 s3.open.viewer.bucket.action=View Bucket
 s3.open.viewer.bucket.failed=Failed to open bucket
+s3.open.viewer.bucket_does_not_exist=Bucket has been deleted.
```

Comment:
`Bucket {0} does not exist`?

Reply:
Yes, this sounds better. Can we display the error as `Bucket {0} does not exist/has been deleted`?